slowfast/models/video_model_builder.py

```diff
@@ -585,12 +600,12 @@ def _construct_network(self, cfg):
             )
 
     def forward(self, x, bboxes=None):
-        x = self.s1(x)
-        x = self.s2(x)
+        x = self.s12(x)
```

Comment:
Flagging here that this change will break the backward compatibility of checkpoint loading (pretrain weights loading)